README.md

```diff
@@ -5,3 +5,13 @@ L3AF Architecture Documentation and Discussion
 ## Documentation
 
 - [Release process](https://github.com/l3af-project/governance/blob/main/docs/RELEASE_PROCESS.md)
+- [L3AF Website](https://l3af.io/): Project landing page
+- [Code](https://github.com/l3af-project): Source Code
+- [Contributing](https://github.com/l3af-project/l3afd/blob/main/docs/CONTRIBUTING.md): Contributor Guide
+- [Wiki](https://wiki.lfnetworking.org/x/8AADAw): Meeting notes and project team resources/updates
+- [Mail](https://lists.l3af.io/g/main): Connect here by mail
```

Comment:
Can you change the email link to [Mail](main@lists.l3af.io)

Reply:
I have made the changes, kindly check.